### PR TITLE
Utility class for positioning images at the top and bottom

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Created an image-position utility to stick an image to the top or bottom of a selected parent container. 

## QA
- Run `gulp jekyll`
- Check the examples:
 - http://127.0.0.1:4001/vanilla-brochure-theme/examples/utilities/image-position/top/
 - http://127.0.0.1:4001/vanilla-brochure-theme/examples/utilities/image-position/bottom/
- Check the documentation is ok
- Check over the code

## Details
Fixes https://github.com/ubuntudesign/vanilla-brochure-theme/issues/6

### CODE REVIEW

**Comment by barrymcgee** [comment]
@anthonydillon Given this is just top/bottom, have you discussed the [rest of the spec](https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Image%20positioning/image-positioning.png) with @yaili?

**Comment by anthonydillon** [comment]
No I haven't, I thought we were just going for top and bottom due to full width designs. Will catch up with @yaili. Thanks.

**Comment by anthonydillon** [comment]
Added bi-directional positions such as `top left`, `bottom right`, etc. It is not trivial to add the stretching image utilities and feel it is likely the best solution would be to use a background image. Think these extra utilites could come in a follow up branch if required.

File: docs/utilities/_image-position.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+collection: utilities
+title: Image position
+---
+
+Image position is a utility to position an image to the top or bottom of a
```
Comment (barrymcgee) [inline docs/utilities/_image-position.md]: Change to: "Image position is a utility to position an image _within_ a
 parent container."

**Comment by anthonydillon** [comment]
Thanks @barrymcgee. Updated the docs with the suggestion and a comment about only affecting medium and large screen sizes. My guess is we only want absolute positioning on medium and large screens as the stacking will cause images to overlay text.

**Pull request merged**